kubernetes/samples/scripts/create-weblogic-domain/domain-home-in-image/create-domain.sh

```diff
@@ -0,0 +1,418 @@
+#!/usr/bin/env bash
```

Comment:
This script shares most of the functions with the scripts in domain-home-on-pv sample. We need to find a way to most the shared functions into a util instead of duplicate them. But I am okay with getting this in first and fix it later.